### ISSUE TITLE
Suggest a better title

### ISSUE THREAD

**Issue by terencehuynh** [terencehuynh/alp-fail#7]
Pls help 🙏

**Comment by auscompgeek** [comment]
I feel removing the title altogether for now would be a good step in the right direction. I'm personally not a huge fan of the tone it creates in comparison to the contents of the letter.

**Comment by mattcen** [comment]
I agree the current title sets a poor tone. It's also ableist.

How about: "What have you done."
(No question mark; a statement of incredulity that's more rhetorical than anything.)

**Comment by timbertson** [comment]
Yeah, definitely needs changing. I'm 👍 on just removing the title (leaving the existing subheader). Failing that, @mattcen 's idea sounds good (although I'd use a question mark 🤷‍♂️ )

**Comment by terencehuynh** [comment]
Emoji? Like this one >> 😡

**Comment by terencehuynh** [comment]
If people are happy with removing the title now (we kinda got the point across that with that...), Will do so.

We do need a new open graph image too...

**Comment by mattcen** [comment]
Yeah, let's start by removing it while we discuss alternatives.

This could work:

> What have you done, Labor? 😡

I dunno how open graph images work.

**Comment by auscompgeek** [comment]
I think the Open Graph image is in the repo here: https://github.com/terencehuynh/alp-fail/blob/master/src/components/ogimage.jpg

Replacing it should do the trick, if I'm reading the build code correctly.

**Comment by terencehuynh** [comment]
Well, someone just used the title on their headline. 

https://www.businessinsider.com.au/you-bunch-of-idiots-australias-tech-industry-savages-labor-for-backing-the-governments-encryption-bill-2018-12

**Comment by mattcen** [comment]
Oh dear :(
OK, how does #136 look?

**Comment by willhackett** [comment]
I think the title is spot on.
